features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/BusinessInformation/BusinessInformation.jsx

```diff
@@ -88,6 +96,18 @@ class BusinessInformation extends React.Component {
         } else if (name === 'responseCaching') {
             value = checked ? 'Enabled' : 'Disabled';
         }
+        // Validate whether mandatory fields are filled and given valid email addresses.
+        if (name === 'businessOwner') {
+            this.isValidBusinessOwnerName = (value !== '');
+        }
+        if (name === 'businessOwnerEmail') {
+            const emailStatus = Joi.string().email({ tlds: true }).required().validate(value).error;
```

Comment:
Shall we add these validations in [APIValidation.js](https://github.com/wso2/carbon-apimgt/blob/master/features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/data/APIValidation.js) and use, as we are having the remaining validations in this single place.
Related issue https://github.com/wso2/product-apim/issues/5391

Reply:
Added.